packages/flutter/lib/src/material/stepper.dart

```diff
@@ -406,9 +454,11 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
 
     final ThemeData themeData = Theme.of(context)!;
     final ColorScheme colorScheme = themeData.colorScheme;
-    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
+    final MaterialLocalizations localizations =
+        MaterialLocalizations.of(context)!;
```

Comment:
I think you can remove the null operator in this line because the `of` method is expected to never return null.

Reply:
But it can't pass the test because of these null
```
flutter test --no-null-assertions --enable-experiment=non-nullable test/stepper_test.dart
```
I add null operator as the test error described